update SQL Server README (#138)
hasura/ndc-sqlserver: README.md

Finally, now that our `DataConnectorLink` has the correct environment variables configured for the SQL Server connector,
we can run the update command to have the CLI look at the configuration JSON and transform it to reflect our database's

> schema in `hml` format. In a new terminal tab from the root of your project, run:

Reviewer: ```suggestion
schema in `hml` format. In a new terminal tab from your project's root directory run:
```

### 8. Import _all_ your indices

> You can do this in one convenience command. From the root of your project, run:

Reviewer: ```suggestion
You can do this with just one command. From your project's root directory, run:
```

### 9. Create a supergraph build

Pass the `local` subcommand along with specifying the output directory as `./engine` in the root of the project. This

> directory is used by the docker-compose file to serve the engine locally. From the root of your project, run:

Reviewer: ```suggestion
directory is used by the docker-compose file to serve the engine locally. From your project's root directory, run:
```
